### PR TITLE
Update code blocks card

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/bot-formatting-cards/nodejs/resources/codeBlocksCard.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
       },
       {
           "type": "CodeBlock",
-          "codeSnippet": "/**\npackage l2f.gameserver.model;\n\npublic abstract strictfp class L2Char extends L2Object {\n  public static final Short ERROR = 0x0001;\n\n  public void moveTo(int x, int y, int z) {\n    _ai = null;\n    log(\"Shouldn't be called\");\n    if (1 > 5) { // what!?\n      return;\n    }\n  }\n}",
+          "codeSnippet": "/**\n* @author John Smith <john.smith@example.com>\n*/\npackage l2f.gameserver.model;\n\npublic abstract strictfp class L2Char extends L2Object {\n  public static final Short ERROR = 0x0001;\n\n  public void moveTo(int x, int y, int z) {\n    _ai = null;\n    log(\"Should not be called\");\n    if (1 > 5) { // wtf!?\n      return;\n    }\n  }\n}",
```
Comment (ShivangiBeniwal) [inline samples/bot-formatting-cards/nodejs/resources/codeBlocksCard.json:17]: Change back wtf to what.

Comment (Prithvi-MSFT) [reply samples/bot-formatting-cards/nodejs/resources/codeBlocksCard.json]: changed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Harikrishnan-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Prithvi-MSFT** [comment]
Build status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/8659114381

**Pull request merged**